pandas-stubs/core/strings.pyi

```diff
@@ -1,105 +1,171 @@
-    def decode(self, encoding, errors: str = ...) -> T: ...
-    def encode(self, encoding, errors: str = ...) -> T: ...
-    def strip(self, to_strip=...) -> T: ...
-    def lstrip(self, to_strip=...) -> T: ...
-    def rstrip(self, to_strip=...) -> T: ...
-    def wrap(self, width, **kwargs) -> T: ...
-    def get_dummies(self, sep: str = ...) -> T: ...
-    def translate(self, table) -> T: ...
-    count = ...
-    startswith = ...
-    endswith = ...
-    findall = ...
-    def extract(self, pat, flags: int = ..., expand: bool = ...) -> T: ...
-    def extractall(self, pat, flags: int = ...) -> T: ...
-    def find(self, sub, start: int = ..., end=...) -> T: ...
-    def rfind(self, sub, start: int = ..., end=...) -> T: ...
-    def normalize(self, form) -> T: ...
-    def index(self, sub, start: int = ..., end=...) -> T: ...
-    def rindex(self, sub, start: int = ..., end=...) -> T: ...
-    len = ...
-    lower = ...
-    upper = ...
-    title = ...
-    capitalize = ...
-    swapcase = ...
-    casefold = ...
-    isalnum = ...
-    isalpha = ...
-    isdigit = ...
-    isspace = ...
-    islower = ...
-    isupper = ...
-    istitle = ...
-    isnumeric = ...
-    isdecimal = ...
+    def repeat(self, repeats: int | Sequence[int]) -> T: ...
+    def pad(
+        self,
+        width: int,
+        side: Literal["left", "right", "both"] = ...,
+        fillchar: str = ...,
+    ) -> T: ...
+    def center(self, width: int, fillchar: str = ...) -> T: ...
+    def ljust(self, width: int, fillchar: str = ...) -> T: ...
+    def rjust(self, width: int, fillchar: str = ...) -> T: ...
+    def zfill(self, width: int) -> T: ...
+    def slice(
+        self, start: int | None = ..., stop: int | None = ..., step: int | None = ...
+    ) -> T: ...
+    def slice_replace(
+        self, start: int | None = ..., stop: int | None = ..., repl: str | None = ...
+    ) -> T: ...
+    def decode(self, encoding: str, errors: str = ...) -> T: ...
+    def encode(self, encoding: str, errors: str = ...) -> T: ...
+    def strip(self, to_strip: str | None = ...) -> T: ...
+    def lstrip(self, to_strip: str | None = ...) -> T: ...
+    def rstrip(self, to_strip: str | None = ...) -> T: ...
+    def wrap(
+        self,
+        width: int,
+        expand_tabs: bool | None = ...,
+        replace_whitespace: bool | None = ...,
+        drop_whitespace: bool | None = ...,
+        break_long_words: bool | None = ...,
+        break_on_hyphens: bool | None = ...,
+    ) -> T: ...
+    def get_dummies(self, sep: str = ...) -> pd.DataFrame: ...
+    def translate(self, table: dict[int, int | str | None] | None) -> T: ...
+    def count(self, pat: str, flags: int = ...) -> Series[int]: ...
+    def startswith(self, pat: str, na: Any = ...) -> Series[bool]: ...
+    def endswith(self, pat: str, na: Any = ...) -> Series[bool]: ...
+    def findall(self, pat: str, flags: int = ...) -> Series[Any]: ...
```

Comment:
Since it is a `Series` of objects, just return `Series` instead of `Series[Any]` and then the test that you commented out will work.

Reply:
Fixed